build.gradle

```diff
@@ -178,7 +178,9 @@ sonarqube {
                                      "src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/infrastructure/SystemDateProvider.java," +
                                      "src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/Application.java"
         property "sonar.cpd.exclusions", "src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/entities/AsylumCase.java," +
-                                         "src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/entities/AsylumCaseBuilder.java"
+                                         "src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/entities/AsylumCaseBuilder.java," +
```

Comment:
I think we can remove first two classes from configuration `AsylumCase` and `AsylumCaseBuilder`

Reply:
@raghera Are you ok with this changes? I like this exclusion as Personalisation classes are 1 to 1 mappings with notification templates and they are quite small and readable in my opinion.